app/controllers/users/registrations_controller.rb

```diff
@@ -49,23 +40,17 @@ def email_domain
     @email.split("@")[1]
   end
 
-  def handle_matching_schools
-    unclaimed_schools = @schools.filter { |school| school.induction_coordinator_profiles.none? }
-
-    if unclaimed_schools.any?
-      redirect_to controller: "users/registrations", action: :confirm_school, school_ids: unclaimed_schools, email: @email
-    else
-      redirect_to root_path, alert: "Someone from your school has already signed up"
-    end
+  def render_start_registration
+    flash.now[:alert] = "Your details did not match any schools."
+    render :start_registration
```

Comment:
I don't know if we want to get into pattern of flash messages - do we render any validation on `start_registration`? Could we add an error there, that will show in error summary?

Reply:
No we are not rendering any validation errors at the moment. The form has school as well as  user information and we are running checks on the relationship between these two and rendering the correct template based on that. This one is the only error message I think the rest are just information pages. Our designer was quite adamant that I can't replace these pages with error / flash messages as the two are fundamentally serving different purposes.

Reply:
Well, yeah, I agree with not removing pages.

What I was thinking about was showing the same `"Your details did not match any schools."` message in form of a form error, rather than a flash message. Do you think that would be possible?

Reply:
I think what you describe is better for model validations rather than when checking connections between models like we are doing here. I'm sure it can be done. One way would be to define a new entity that has a user and a school and does the checks. I just don't think it would be worth the cost.

Reply:
Alright, flash message it is then :)

Reply:
@kicferk1 I did a demo of this with our designers and they will be splitting this into two screens so we can validate a single entity. That should address the error messaging concerns you had but it may be part of another pr.

Reply:
Thanks for following up on it, sounds good to me to make it a separate PR.